feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -192,12 +201,11 @@ func TestBasic(t *testing.T) {
 	// Allow 1s of lag between adjacency appearing and all data being populated
```

Comment:
Again, comment dont seem to match what you are proposing here. Can you please update?

Reply:
updated , Thank you.